applications/launchpad/gui-react/src/components/Onboarding/ProgressIndicator/index.tsx

```diff
@@ -4,29 +4,44 @@ import { StyledContainer } from './styles'
 import { ProgressIndicatorProps } from './types'
 
 const ProgressIndicator = ({ overallFill }: ProgressIndicatorProps) => {
-  const [fillOne, setFillOne] = useState<number | undefined>(0)
-  const [fillTwo, setFillTwo] = useState<number | undefined>(0)
-  const [fillThree, setFillThree] = useState<number | undefined>(0)
-  const [fillFour, setFillFour] = useState<number | undefined>(0)
+  const [fillOne, setFillOne] = useState<number | undefined>(0.0001)
```

Comment:
I'd move 0.0001 to a const somewhere, which would explain why its not 0
REALLY_CLOSE_TO_ZERO_BECAUSE_XYZ

Reply:
Done!